src/App.js

```diff
@@ -11,17 +11,21 @@ import LoginSuccess from "./pages/login-success/LoginSuccess";
 function App() {
   const location = useLocation();
 
+  const initializeAppForLoggedInUser = async (token) => {
+    await rootStore.userStore.fetchUserInformation(token);
```

Comment:
Step 4: Last, we need to replace all of the content we previously had in `initializeAppForLoggedInUser` to calling the generator function we created in the user store.

That's it! Now we cleaned this function up a bunch, it should be easier to add other fetch calls in here now and boot the app!